scipy/spatial/qhull.pyx

```diff
@@ -2433,6 +2505,10 @@ class Voronoi(_QhullUser):
         Index of the Voronoi region for each input point.
         If qhull option "Qc" was not specified, the list will contain -1
         for points that are not associated with a Voronoi region.
+    furthest_site
+        True if this was a furthest site tesselation and False if not.
+
+        .. versionadded:: 1.3.x
```

Comment:
It looks like the git history / changes got mangled a bit -- this stuff has creeped back in now.